props_4_ccache.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <!-- /Z7 of cl.exe, ref: https://learn.microsoft.com/en-us/cpp/build/reference/z7-zi-zi-debug-information-format -->
+      <DebugInformationFormat>OldStyle</DebugInformationFormat>
+      <ObjectFileName>$(IntDir)%(FileName).obj</ObjectFileName>      
+    </ClCompile>
+  </ItemDefinitionGroup>
+</Project>
```

Comment:
Could you add an empty line at the bottom?

Also, I think a better place for this file would be in `tools/msvs` as there are all Windows-specific tools. Once you move it, just make sure to change the path in `ForceImportAfterCppProps` in `vcbuild.bat` too.

Reply:
ok, got it :)